pkg/scheduler/apis/config/v1/conversion.go

```diff
@@ -105,3 +105,7 @@ func convertToExternalPluginConfigArgs(out *v1.KubeSchedulerConfiguration) error
 	}
 	return nil
 }
+
+func Convert_config_KubeSchedulerProfile_To_v1_KubeSchedulerProfile(in *config.KubeSchedulerProfile, out *v1.KubeSchedulerProfile, s conversion.Scope) error {
```

Comment:
We have to explicitly add it in `conversion.go`  because `make update` will delete it with a warning. See my discussions with @Huang-Wei  https://github.com/kubernetes/kubernetes/pull/112521/files#r974720161

Reply:
You are right. We can just used the auto-generated one.